docs/fast_data/faq/bootstrapping-fast-data.md

```diff
@@ -0,0 +1,154 @@
+| Kafka     | 4 brokers      | 600.000 nominal events X day, i.e. 30GB of ingestion messages. |
+| MongoDB   | 3 replica sets | Enterprise distribution having 32 GB of RAM.                   |
+| Fast Data | 6 nodes        | K8S cluster with each node having 8 core.                      |
+
+## Kafka
+
+
+### What is the topic setup I should use to guarantee an high rate of ingestion?
+
+
+When creating and managing topics in a Kafka cluster, is crucial to define a number of partitions that can balance the identifiers of messages among different replicas that can read messages from the topic with the same consumer group. 
+
+While designing your Fast Data solution, is crucial that <ins>the message producer will assign the same key to events involving the same record</ins>: in this way, they will be assigned always to the same partition and then processed always in the same order.  
+
+:::tip Useful Resources
+For more detailed benchmarks about topic partitions and data distribution among it, you can refer to this [Confluent blog post](https://www.confluent.io/blog/how-choose-number-topics-partitions-kafka-cluster/) .
+:::
+
+### What is the topic setup I should use to guarantee an high availability?
+
+A topic is distributed among the brokers of your cluster, with a parameter called _replication factor_: for critical messages, this value should be set to the number of brokers, meaning that the data of a topic will be stored inside all brokers.
+
+You can also specify how many replicas are needed to send acknowledgments on produced messages before committing, to guarantee the synchronization of the distributed data.
+
+This value can be set at the topic level with the parameter `min.insync.replicas` and is usually equal to `# of brokers - 1`. Critical flows may require a number of in-sync replicas equal to the number of brokers. 
+
+## Mongo Db
+
+### What should be the required resources to perform an Initial Load?
+
+The **Initial Load** is a manual operation made at runtime to initialize the projection's records with the most recent copy of the dataset from the system of record linked to it.
+
+This operation is carried out when we need the more recent copy of a table, in order to transfer the data present within the Fast Data system, with a **Change Data Capture** (**CDC**) and will be then processed by the [Real-Time Updater (RTU)](/fast_data/realtime_updater.md).
+
+:::danger Before executing the Initial Load
+Since an high number of ingestion messages will be handled by the RTU, remember **to disable strategy** during the execution of the Initial Load.
+
+This will avoid to perform additional MongoDb queries made by the strategies to generate projection changes records.
+:::
+
+:::tip Execution Time
+An initial load of `6.000.000 records` took around _10 minutes_, which is approximately a rate of `60k message/minute`: for this message rate, up until 70% of CPU from a single MongoDb core was used.  
```

Comment:
Should we specify here how many replicas of RTU were deployed?

Please note that when Projection Storer will be available this documentation has to be updated to account also for it.

Reply:
hi @danibix95 i've added this information